alembic/versions/0f36b3ad81fc_add_third_party_user_and_third_party_.py

```diff
@@ -13,7 +13,10 @@
 from alembic import op
 import sqlalchemy as sa
 from sqlalchemy_enum34 import EnumType
-from compair.models import ThirdPartyType
+from enum import Enum
+
+class ThirdPartyType(Enum):
```

Comment:
why do you duplicate the class here?

Reply:
Its something that I overlooked before when adding the enums to the migrations. Using references to the live, changeable classes can potentially break upward and downward migrations a certain steps.

Ex: if there was a really old ComPAIR db somewhere that didn't hadn't run `0f36b3ad81fc_add_third_party_user_and_third_party_.py` yet, it would technically add an enum with `cas` instead of the correct `cwl` (which I later renamed to `cas` in another migration).

Reply:
Also technically should be in a different pr but i was already dealing with the migrations because of uuids and remembered that this should be done (was thinking about the eventual shibboleth changes to this column